Assets/Plugins/Android/Core/AndroidPlatformSpecifics.cs

```diff
@@ -30,24 +30,36 @@
 using JavaVM = System.IntPtr;
 using System.Diagnostics;
 
-#if UNITY_ANDROID && !UNITY_EDITOR
+#if UNITY_ANDROID 
 [assembly: AlwaysLinkAssembly]
 namespace PlayEveryWare.EpicOnlineServices
 {
+    using Epic.OnlineServices.Platform;
+
+    public class EOSCreateOptions
+    {
+        public Options options;
+    }
+
+    public class EOSInitializeOptions
+    {
+        public AndroidInitializeOptions options;
+    }
+
     //-------------------------------------------------------------------------
     // Android specific Unity Parts.
-    public class EOSPlatformSpecificsAndroid : PlatformSpecifics<AndroidConfig>
+    public class AndroidPlatformSpecifics : PlatformSpecifics<AndroidConfig>
```

Comment:
I would argue that as a hotfix this should not rename any classes unless it's essential that they be renamed.

Reply:
Yep! Good catch; I did this out of habit.